Ruddarr/Views/MoviesView.swift

```diff
@@ -8,6 +8,8 @@ struct MoviesView: View {
     @State private var fetchedMovies = false
     @State private var sort: MovieSort = .init()
 
+    // TODO: this should be StateObject if we're creating it here.
+    // Maybe we can switch to @Observable and not worry about this?
     @ObservedObject var movies = MovieModel()
```

Comment:
@pteasima I don't understand, can you do whatever is best here?

Reply:
This was a potential bug, we would get  a new instance of MovieModel everytime the parent view re-rendered (https://www.avanderlee.com/swiftui/stateobject-observedobject-differences/) , `@Observable` is the new macro replacing `ObservableObject` altogether. I'll see if its easy enough to migrate to that.